modules/ROOT/pages/troubleshooting-visualization.adoc

```diff
@@ -0,0 +1,87 @@
+= Troubleshooting Visualization
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+3
+
+== About the Troubleshooting Visualization
+
+The Troubleshooting Visualization shows you the current and past health of your Mule apps. It allows you to quickly visualize where your network is slow, receiving heavy traffic or failing. From this visualization, you can drill down directly into Anypoint Monitoring dashboards or logs to continue to troubleshoot the performance of your systems.
+
+== Choosing your Metric
+
+Use the options in the *Metric* card to select the metric that you want to show for all the nodes in your application network. For applicable metrics, the metric value will also appear on the connection, or _edge_.
```

Comment:
Wordy? How about:
```suggestion
Select options in the *Metric* card to show metrics for all the nodes in your application network. For applicable metrics, these values also appear on the connection, or _edge_.
```

Reply:
Does the user need to know about these being on the connection/edge at this point in the doc?